### DISCUSSION TITLE
How to get a memory RGBRGBRGB... or RGBARGBARGBA... generated plot image instead of saving it to disk ?

### DISCUSSION THREAD

**Discussion by tomay3000** [alandefreitas/matplotplusplus#175] [Q&A]
Hello,
as I am using `wxWidgets` as my main GUI, I need to get the generated plot image as `RGBRGBRGB...` or `RGBARGBARGBA...` formats, so I can display it in a `wxStaticBitmap` inside a `wxPanel` instead of saving it into the hard disk.

TIA.

**Comment by alandefreitas** [comment]
There's no backend that does that at the moment.

**Reply by tomay3000** [reply]
Thank you.
I hope it will be implemented the soon future.
